arches_for_science/media/js/views/components/workflows/project-report-workflow/report-template-select.js

```diff
@@ -19,7 +19,7 @@ define([
         this.getTemplates = async() => {
             const response = await fetch(arches.urls.reports_list);
 
-            if(response.ok) {
+            if (!response.ok) {
```

Comment:
I think this should remain `if(response.ok)`. It looks like this might be causing the unhandled promise error.